index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link ref="stylesheet" href="./style.css">
```

Comment:
I believe you have misspelled `ref` here. Instead it should be `rel`. This error is causing your CSS file not to load. Kindly fix this issue.